From a152de2d3c45b083baf8f57a6da0a890074578ae Mon Sep 17 00:00:00 2001
From: Gabriel Nagy <gabriel.nagy@canonical.com>
Date: Mon, 13 May 2024 16:07:25 +0200
Subject: [PATCH] Add magic comment to vendored files

In order for them to be excluded by the TiCS checker.
---
 python/samba/gp/gp_cert_auto_enroll_ext.py | 2 ++
 python/samba/gp/gpclass.py                 | 2 ++
 python/samba/gp/util/logging.py            | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/python/samba/gp/gp_cert_auto_enroll_ext.py b/python/samba/gp/gp_cert_auto_enroll_ext.py
index 9b743cb7f9b..0a590dd660f 100644
--- a/python/samba/gp/gp_cert_auto_enroll_ext.py
+++ b/python/samba/gp/gp_cert_auto_enroll_ext.py
@@ -1,3 +1,5 @@
+# TiCS: disabled # generated by CI auto-update workflow
+
 # gp_cert_auto_enroll_ext samba group policy
 # Copyright (C) David Mulder <dmulder@suse.com> 2021
 #
diff --git a/python/samba/gp/gpclass.py b/python/samba/gp/gpclass.py
index 605f94f3317..afd812e7a25 100644
--- a/python/samba/gp/gpclass.py
+++ b/python/samba/gp/gpclass.py
@@ -1,3 +1,5 @@
+# TiCS: disabled # generated by CI auto-update workflow
+
 # Reads important GPO parameters and updates Samba
 # Copyright (C) Luke Morrison <luc785@.hotmail.com> 2013
 #
diff --git a/python/samba/gp/util/logging.py b/python/samba/gp/util/logging.py
index da085d8d7e6..f748e66b39c 100644
--- a/python/samba/gp/util/logging.py
+++ b/python/samba/gp/util/logging.py
@@ -1,3 +1,5 @@
+# TiCS: disabled # generated by CI auto-update workflow
+
 #
 # samba-gpupdate enhanced logging
 #
-- 
2.44.0

